### ISSUE TITLE
DatArchive.fork is not a function

### ISSUE THREAD

**Issue by ar-naseef** [beakerbrowser/node-dat-archive#6]
DatArchive.fork throws an error.
 - [DatArchive.fork API ](https://beakerbrowser.com/docs/apis/dat.html#datarchive-fork)

This seems not implemented.

**Comment by pfrazee** [comment]
Yeah, [not yet implemented](https://github.com/beakerbrowser/node-dat-archive#differences-from-browser-api). PRs accepted.

**Comment by ar-naseef** [comment]
Yeah, I'll make a PR.

**Comment by ar-naseef** [comment]
https://github.com/beakerbrowser/node-dat-archive/pull/8
